extensions/import/src/test/wizard/api/basePage.test.ts

```diff
@@ -59,12 +59,15 @@ describe('import extension wizard pages', function () {
 
 		let serverValues = await importPage.getServerValues();
 
+		// getServer value should be undefined for null active connections
 		should(serverValues).undefined();
 
 		// mocking getActive connection returns empty array
 		mockApiWrapper.setup(x => x.getActiveConnections()).returns(async () => { return [] as azdata.connection.Connection[]; });
 
 		serverValues = await importPage.getServerValues();
+
+		// getServer value should be undefined for empty active connections
 		should(serverValues).undefined();
```

Comment:
While you're here - these checks in this test should have a message saying which value they're checking that failed. If one was to fail we'd have no way of knowing unless we looked at the actual javascript or debugged into it.